CONTRIBUTING.md

```diff
@@ -13,6 +13,9 @@ There are a few basic ground-rules for contributors:
 1. **External API changes and significant modifications** ought to be subject to an **internal pull-request** to solicit feedback from other contributors.
 1. Internal pull-requests to solicit feedback are *encouraged* for any other non-trivial contribution but left to the discretion of the contributor.
 1. Contributors should attempt to adhere to the prevailing code-style.
+1. At least two contributors, or one core member, must approve pull-requests prior to merging.
```

Comment:
I wouldn't make differences between core and community members. It should based on the project experience and of the knowledge of the developer. I would just say: At least two members are required to approve a PR. For none code changes e.g documentation it's enough to have one approval of a member. Significant architectural changes should be announced to the whole team and need at least one approval of a team member who has same understanding of the change like you.

Reply:
I think we should live with this and try out how it goes. It is significantly different than what we do in node core, and I'm 👍  to experiment!